Update existing evaluation pages (#1180)
ACCESS-Hive/access-hive.github.io: docs/model_evaluation/evaluation_on_gadi/model_live_diagnostics.md

## What is Model Live Diagnostics?

> _Model Live Diagnostics_ is a simple, accessible and easy-to-use Jupyter notebook-based framework for the ACCESS modelling community to monitor, visualise and evaluate the behaviour of models in real time (_live_) while they run on _Gadi_.

Reviewer: A few too many adjectives
```suggestion
_Model Live Diagnostics_ is a simple, accessible and easy-to-use framework for the ACCESS modelling community to monitor, visualise and evaluate the behaviour of models in real time (_live_) while they run on _Gadi_.
```

Author: ```suggestion
_Model Live Diagnostics_ is an accessible Jupyter notebook-based framework for the ACCESS modelling community to monitor, visualise and evaluate the behaviour of models in real time (_live_) while they run on _Gadi_.
```

## What is Model Live Diagnostics?

> _Model Live Diagnostics_ is an accessible Jupyter notebook-based framework for the ACCESS modelling community to monitor, visualise and evaluate the behaviour of models in real time (_live_) while they run on _Gadi_.

Reviewer: ```suggestion
_Model Live Diagnostics_ is an easily-accessible Jupyter-notebook-based framework for the ACCESS modelling community to monitor, visualise and evaluate the behaviour of models in real time (_live_) while they run on _Gadi_.
```

In addition to monitoring a live model, the package also provides the functionality to load, visualise and compare legacy ACCESS model data with the live model.

???+ warning "Support Level: Supported on _Gadi_, not in active development"

>     Model Live Diagnostics Tool is not in active development and can be used on _Gadi_ provided 3rd party packages are compatible.

Reviewer: What does this mean? Which 3rd party packages?

Author: its similar to the wording in cosima cookbook ..as it's not in active development I don't think we can guarantee its always working with dependencies in the conda environment. Not sure if its docs are up to date, may need to check this

Reviewer: But if it's included in the `conda/analysis3` environments it should at least work, otherwise what's the point of including it in the first place?

Author: i've removed, im not sure its in analysis3, i think it just needed some wording around not being as maintained

    <img src="../../../assets/model_evaluation/live_diagnostics/tutorial_image_4.png" alt="Plot of total liquid seawater mass over time of the ‘live’ ACCES CM2 run." width="75%"/>
</div>

> The plot above shows the users own 'live' data so doesn't have a name. It is also possible to load and compare _legacy data_, such as other previous ACCESS-CM2 model runs (_by578_ and _by578a_ etc.)

Reviewer: ```suggestion
The plot above shows the user's own _live_ data, which has no name assigned. It is also possible to load and compare _legacy data_, such as outputs from previous ACCESS-CM2 model runs (e.g., _by578_ and _by578a_).
```
